cluster/examples/kubernetes/ceph/filesystem-ec.yaml

```diff
@@ -42,7 +42,17 @@ spec:
     #  - key: mds-node
     #    operator: Exists
     #  podAffinity:
-    #  podAntiAffinity:
+       podAntiAffinity:
+          requiredDuringSchedulingIgnoredDuringExecution:
+          - labelSelector:
```

Comment:
Is there a simpler way to express the label selector such as this?
```
    - labelSelector:
        app: rook-ceph-mds
```

Reply:
Yes. Done.

Reply:
This did not work. Had to switch back to the previous format.